nntrainer/tensor/swap_device.h

```diff
@@ -126,6 +128,31 @@ class SwapDevice {
    */
   unsigned int getNumLoadedTensors();
 
+  /**
+   * @brief Check if address is unmapped or not.
+   *
+   * @param address
+   * @return bool true if address is unmapped, otherwise false.
+   */
+  bool address_unmapped(void *address) {
```

Comment:
how about modifying the function name like `is_address_unmapped`?

Reply:
Good idea!, thank you for review